### ISSUE TITLE
WriteElement() in IaXmlWriter.cs

### ISSUE THREAD

**Issue by AndrewJamesMills** [Intacct/intacct-sdk-net#99]
The WriteElement() in IaXmlWriter.cs has many overloads.  e.g.

```csharp
public void WriteElement(string localName, string value, bool writeNull = false)
public void WriteElement(string localName, int? value, bool writeNull = false)
public void WriteElement(string localName, decimal? value, bool writeNull = false)
```

If I have a custom field that's actually null this method would be called as below based on its implementation using something like ProjectCreate()
```csharp
WriteElement("CustomFielName", NULL, true)
```

Since that's always hard-coded to write null fields, the WriteElement signature with a null is ambiguous between the three WriteElement() examples above.

**Comment by intacctcoleenho** [comment]
This is going to be fixed in the next minor release.

**Comment by intacctcoleenho** [comment]
Now available in v2.3.0
